app/models/repository/git.rb

```diff
@@ -41,11 +41,49 @@ def self.human_attribute_name(attribute_key_name, options = {})
   end
 
   def self.scm_adapter_class
-    Redmine::Scm::Adapters::GitAdapter
+    OpenProject::Scm::Adapters::Git
   end
 
-  def self.scm_name
-    'Git'
+  def configure(args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `configure(*)` if you want the method to accept any arguments but don't care about them.